toppingmaker/exportsettings.py

```diff
@@ -58,9 +66,12 @@ class ToppingType(Enum):
         SOURCE = 3
 
     def __init__(self):
+        # layertree settings
         self.qmlstyle_setting_nodes = {}
         self.definition_setting_nodes = {}
         self.source_setting_nodes = {}
+        # maptheme settings
+        self.mapthemes = []
```

Comment:
This is defined but not used, or am I missing something?

Reply:
Yes. It's just a list. Accessible from the outside.

Reply:
What for? Is there some interface between this list and internals of this class?

Reply:
On exporting it's used here: https://github.com/opengisch/toppingmaker/pull/8/files#diff-f7d3afd72844e3f050663a02a3966033f4dce128634342da1b2d1585e9275608R291

The part where it's set is not yet implemented (will be in the tests of the other draft-pr and in the GUI model of Model Baker Topping Exporter). User lists the map themes of interest.

Does it need a setter and getter function for that? It feels like an overkill...

Reply:
Ah, no, sounds good like that.
Now I wonder if the comment "maptheme settings" can be made a tiny bit more verbose.

Reply:
Added a sentence to the main description of the class as well.